mindsdb/interfaces/jobs/jobs_controller.py

```diff
@@ -0,0 +1,351 @@
+import re
+import datetime as dt
+from dateutil.relativedelta import relativedelta
+
+import torch.multiprocessing as mp
+import sqlalchemy as sa
+
+from mindsdb_sql import parse_sql, ParsingException
+
+from mindsdb.interfaces.storage import db
+from mindsdb.interfaces.database.projects import ProjectController
+from mindsdb.utilities.context import context as ctx
+from mindsdb.utilities import log
+
+mp_ctx = mp.get_context('spawn')
```

Comment:
Shouldn't this be inside a try-except? I think the main entry point to mindsdb is already setting this.

Reply:
I looked at the other examples of mp.get_context, they are not try-catched

Reply:
My bad, I was thinking of `set_start_method`, ignore this)